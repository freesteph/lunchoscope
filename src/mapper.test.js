const data = require("../test/sampleOutput");
const mapper = require("./mapper");

describe("mapper", () => {
  it("correctly maps all days", () => {
    expect(mapper(data)).toMatchSnapshot();
  });
});
