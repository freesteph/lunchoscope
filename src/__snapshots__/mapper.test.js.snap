// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mapper correctly maps all days 1`] = `
Object {
  "friday": Object {
    "meat": Object {
      "description": "e  Battered or breaded fish served with chunky chips, homemade tartare sauce & lemon wedge",
      "price": "£3.70",
    },
    "veg": Object {
      "description": "Spicy mix beans and soy protein tacos wit, served with coriander and tomato salsa Ge and tomatillo rice",
      "price": "£3.70",
    },
  },
  "monday": "N/A",
  "thursday": Object {
    "meat": Object {
      "description": "Grilled Greek style beef burger with beef tomato, red onion, crispy lettuce and whipped feta yoghurt",
      "price": "£3.70",
    },
    "veg": Object {
      "description": "Beetroot and carrot burger with beef ee tomato , crispy lettuce, hash brown and Ge dill and garlic sauce",
      "price": "£3.60",
    },
  },
  "tuesday": Object {
    "meat": Object {
      "description": "British beef and black turtle bean chilli with steamed paprika rice and sour cream",
      "price": "£3.70",
    },
    "veg": Object {
      "description": "Spinach , pea and goat cheese risotto wa ) served with rocket salad",
      "price": "£3.70",
    },
  },
  "wednesday": Object {
    "meat": Object {
      "description": "Chicken, leek and mushroom mac n’ cheese served with chef's salad",
      "price": "£3.70",
    },
    "veg": Object {
      "description": "Portobello mushroom, leek and cherry Go ven tomato mac n cheese served with chef's salad",
      "price": "£3.60",
    },
  },
}
`;
